content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,806 @@
+
+```yaml
+# clusterissuer-lets-encrypt-staging.yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: $EMAIL_ADDRESS
+    privateKeySecretRef:
+      name: letsencrypt-staging
+    solvers:
+    - dns01:
+        azureDNS:
+          resourceGroupName: $AZURE_DEFAULTS_GROUP
+          subscriptionID: $AZURE_SUBSCRIPTION_ID
+          hostedZoneName: $DOMAIN_NAME
+          environment: AzurePublicCloud
+          managedIdentity:
+            clientID: $USER_ASSIGNED_IDENTITY_CLIENT_ID
+```
+🔗 <a href="clusterissuer-lets-encrypt-staging.yaml">`clusterissuer-lets-encrypt-staging.yaml`</a>
+
+
+As you can see there are some variables in the `clusterissuer-lets-encrypt-staging.yaml` which need to be filled in before we apply it;
+most have been defined earlier in this tutorial but you need to set the following:
+
+```bash
+export EMAIL_ADDRESS=<email-address> # ❗ Replace this with your email address
+export AZURE_SUBSCRIPTION=<your-subscription-or-billing-account>  # ❗ Replace this with your Azure account name
+```
+
+Now use `envsubst` to fill in the variables and pipe it into `kubectl apply`, as follows:
+
+```bash
+export AZURE_SUBSCRIPTION_ID=$(az account show --name $AZURE_SUBSCRIPTION --query 'id' -o tsv)
+envsubst < clusterissuer-lets-encrypt-staging.yaml | kubectl apply -f  -
+```
+
+You can check the status of the ClusterIssuer:
+
+```bash
+kubectl describe clusterissuer letsencrypt-staging
+```
+
+Example output
+
+```console
+Status:
+  Acme:
+    Last Registered Email:  firstname.lastname@example.com
+    Uri:                    https://acme-staging-v02.api.letsencrypt.org/acme/acct/77882854
+  Conditions:
+    Last Transition Time:  2022-11-29T13:05:33Z
+    Message:               The ACME account was registered with the ACME server
+    Observed Generation:   1
+    Reason:                ACMEAccountRegistered
+    Status:                True
+    Type:                  Ready
+```
+
+> ℹ️ Let's Encrypt uses the Automatic Certificate Management Environment (ACME) protocol
+> which is why the configuration above is under a key called `acme`.
+>
+> ℹ️ The email address is only used by Let's Encrypt to remind you to renew the certificate after 30 days before expiry. You will only receive this email if something goes wrong when renewing the certificate with cert-manager.
+>
+> ℹ️ The Let's Encrypt production issuer has [very strict rate limits](https://letsencrypt.org/docs/rate-limits/).
+> When you're experimenting and learning, it can be very easy to hit those limits. Because of that risk,
+> we'll start with the Let's Encrypt staging issuer, and once we're happy that it's working
+> we'll switch to the production issuer.
+>
+> 📖 Read more about [configuring the ACME Issuer](../../configuration/acme).
```

Comment:
suggestion: I don't think this link follows https://github.com/cert-manager/website/#task-linking-to-other-documentation-pages